src/libraries/Native/Unix/System.Globalization.Native/CMakeLists.txt

```diff
@@ -80,14 +80,15 @@ if (GEN_SHARED_LIB)
     install_with_stripped_symbols (System.Globalization.Native PROGRAMS .)
 endif()
 
-add_library(System.Globalization.Native-Static
+add_library(System.Globalization.Native-obj
```

Comment:
A nit - for other libraries, we use `_obj` suffix to indicate object libraries, so it would be nice to stay consistent here.